### ISSUE TITLE
Add section to docs site to address accessibility

### ISSUE THREAD

**Issue by clpetersonucf** [ucfopen/Materia#1484]
Given increased accessibility concerns at UCF, the docs site should be a resource to assist both users and developers. For instance:

For users & stakeholders:
- Addressing accessibility status of widgets and the system at large
- Addressing how to assess accessibility status for individual widgets
- Accessibility best practices when implementing Materia

For developers:
- Implementing accessibility best practices when authoring new widgets, including keyboard navigation
- Recommendations for specific accessibility implementations related to: tab ordering, focusable input navigation, tab navigation for components that include multiple inputs, arrow key navigation, keyboard shortcuts, live regions, and considerations for mapping a visually dynamic interface into a sensible screenreader navigable layout.

**Comment by clpetersonucf** [comment]
Duplicate of #1587 and addressed, closing.
